model/device/registry.yaml

```
@@ -3,23 +3,32 @@ groups:
     type: attribute_group
     display_name: Device Attributes
     brief: >
-        Describes device attributes.
+      Describes device attributes.
     attributes:
       - id: device.id
         type: string
         stability: experimental
         brief: >
           A unique identifier representing the device
-        note: >
-          The device identifier MUST only be defined using the values outlined below. This value is not an advertising
-          identifier and MUST NOT be used as such.
-          On iOS (Swift or Objective-C), this value MUST be equal to the [vendor identifier](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor).
-          On Android (Java or Kotlin), this value MUST be equal to the Firebase Installation ID or a globally unique
-          UUID which is persisted across sessions in your application. More information can be found [here](https://developer.android.com/training/articles/user-data-ids)
-          on best practices and exact implementation details.
-          Caution should be taken when storing personal data or anything which can identify a user. GDPR and
-          data protection laws may apply, ensure you do your own due diligence.
-        examples: ['2ab2916d-a51f-4ac8-80ee-45ac31a28092']
+        note: |
+          **The `device.id` SHOULD NOT be used in most user-facing applications due to privacy regulations.
+          Consequently, instrumentations MUST provide it as an OPT-IN feature.**
```

```suggestion
          Consequently, instrumentations that provide it MUST provide it as an opt-in feature.**
```

there is no formal definition of OPT-IN, so let's use lower case